### ISSUE TITLE
Error upon encountering <path> element with float in scientific notation

### ISSUE THREAD

**Issue by ilkupi** [rafaeltonholo/svg-to-compose#56]
Greetings! 

While using this tool to convert from `.svg` I've got an error
> `Not support SVG/Android Vector command. Command = e`

after which tool immediately stops, although there are more files left to process. I'm not using optimisations (run with  `-opt false`)

Upon digging into the problem it looks like there is a problem with parsing floating point `<path>` values written in scientific notation (i.e. `-7.51423e-08` in example below). 
```
<svg width="417" height="247" viewBox="0 0 417 247" fill="none" xmlns="http://www.w3.org/2000/svg">
...
<path id="Rectangle 66291" d="M-7.51423e-08 35.2685C-3.36423e-08 27.2721 6.48233 20.7898 14.4787 20.7898L258.203 20.7898C266.2 20.7898 272.682 27.2721 272.682 35.2685L272.682 172.816C272.682 180.812 266.2 187.295 258.203 187.295L32.577 187.295L-1.03947e-06 221.078L-7.51423e-08 35.2685Z" fill="#D3E7FF"/>
...
</svg>
```
Same notation in <rect> within matrix transform works fine
```
<svg width="225" height="240" viewBox="0 0 225 240" fill="none" xmlns="http://www.w3.org/2000/svg">
...
<rect width="70.6398" height="70.6398" rx="35.3199" transform="matrix(-1 -8.74228e-08 -8.74228e-08 1 123.285 55.6846)" fill="#B61846"/>
...
</svg>
```

**Comment by rafaeltonholo** [comment]
Hi there!

Thank you for mentioning this issue. Parsing the path commands is more complex than parsing the transform, and I forgot to account for scientific notations when parsing the path commands.

I will investigate this as quickly as possible and provide a solution.

Thank you for providing the path that is causing the error; it will be very helpful during debugging.

**Comment by rafaeltonholo** [comment]
Hi @ilkupi,

I have published a fix in the release 1.3.2 for this issue! Could you please check if it fixed the issue? 

If you are using the CLI via `git clone`, please run `git pull` and then run the same command you used, but with the flag `--upgrade` should build the project and apply the fix. In case of just using the script, you can run the same command with the `--upgrade` flag
